### PR TITLE
IoT business takeover

### PR DESCRIPTION
**Pull request by deadlight**
## Done

A new takeover for the IoT whitepaper.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check that the takeover and left-hand takeunder look good on all viewports

### CODE REVIEW

**Comment by joanasa89** [comment]
Looks good!

File: templates/takeovers/_iot-business.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<section class="p-strip--accent is-dark is-deep">
+  <div class="row">
+    <div class="u-equal-height u-vertically-center">
```
Comment (grahambancroft) [inline templates/takeovers/_iot-business.html]: Can these go on the row?

File: templates/takeovers/takeunders/_kubernetes.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<div class="col-6 p-takeunder" style="background-color: #326CE5;">
+  <div class="row">
+    <div class="u-equal-height">
```
Comment (grahambancroft) [inline templates/takeovers/takeunders/_kubernetes.html]: Can this go on the row

**Comment by grahambancroft** [comment]
Takeover image missing at 768px

screen shot 2017-07-18 at 10 30 15

**Comment by deadlight** [comment]
@grahambancroft ready for re-review.

The 768 bug is something I'll make a note of to chase with the Vanilla team.

**Pull request merged**